.github/workflows/ci.yml

```diff
@@ -106,7 +107,7 @@ jobs:
       - build_x86_64_apple_darwin
       - build_x86_64_unknown_linux_gnu
       - build_x86_64_unknown_linux_musl
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
I don't see an issue with this, but why this switch to Ubuntu building here? Seems fine.

Reply:
this is so our `ci.yml`'s deploy (snapshot) is consistency with the `release.yml`'s deploy (the final release)

AKA we would've caught this sooner had we run this on ubuntu

Reply:
Great observation

Reply:
I really wish `mvn deploy` could deploy to 2 `servers` in one go without rebuilding, but this is the best I could figure